File: practice/programiz/sum_of_natural_numbers.c
#include <stdio.h>
/**
 * @brief this function calculates the sum of first n natural numbers
 * @return int sum of natural number
 * @param first_number kis number ko add karana hai
 * @param second_number kis number ko add karana hai
 * @param stop_number kaha tak hame sum add karani hai
 */
int sum_natural_numbers(int number, int stop_number)
{
    int sum = 0;
    for (number = 1; number <= stop_number; number++)
    {
        sum = sum + number;
    }
    return sum;
}

int main(void)
{
    int number;
    int stop_number;
    printf("enter a first number for starting the sum:\n ");
    scanf("%i", &number);
    printf("enter a last number for stopping the sum:\n ");
    scanf("%i", &stop_number);

    // int sum = sum_natural_numbers(number, stop_number);
    printf("%i", sum_natural_numbers(number, stop_number));
    return 0;
}

File: practice/java/SumOfNaturalNumbers.java

import java.util.Scanner;

class SumOfNaturalNumbers {

    /**
     * @brief this function calculate the sum of natural numbers
     * @param number kaha se number ko
     */
    static int sumNumbers(int number, int stop_number) {
        int sum = 0;
        if (number > 0) {
            for (; number <= stop_number; number++) {
                sum = sum + number;
            }
        } else {
            System.out.println("0 is not a natural number plz enter greater than zero number:");
        }
        return sum;

    }

    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number:");
        int number = sc.nextInt();
        System.out.println("Enter the stop number:");
        int stop_number = sc.nextInt();
        int sum = sumNumbers(number, stop_number);
        if (number > 0) {
            System.out.println("Sum of natural numbers from " + number + " to " + stop_number + " is: " + sum);
        }

    }
}
